### DISCUSSION TITLE
Gettings tests to show up in NCrunch

### DISCUSSION THREAD

**Discussion by powerdude** [reqnroll/Reqnroll#90] [Q&A]
Is there anything extra required to have NCrunch recognize, and run, the features?  I just tried to include the quickstart feature and initial binding files and nothing shows in the NCrunch display.  The do show and run in the VS test runner.

**Comment by gasparnagy** [comment]
Yes. NCrunch will provide a full support soon, but until then you need to include the feature files in the "Additional files to include" setting. I have
* Additional files to include: `Features\**.*`

In the config file this looks like this:

```
<ProjectConfiguration>
  <Settings>
    <AdditionalFilesToIncludeForProject>
      <Value>Features\**.*</Value>
    </AdditionalFilesToIncludeForProject>
    <ConsiderInconclusiveTestsAsPassing>True</ConsiderInconclusiveTestsAsPassing>
  </Settings>
</ProjectConfiguration>
```

Also after adding new feature files, you need to build the project at least once and sometimes you also need to reload the project in NCrunch.
